src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+/**
+ * Deals with making sense of the user command
```

Comment:
```suggestion
 * Deals with making sense of the user command.
```
Shouldn't JavaDoc summaries end with a "."?